### ISSUE TITLE
Visibility of event in shared calendars property shouldn't be displayed in read-only mode

### ISSUE THREAD

**Issue by tcitworld** [nextcloud/calendar#1587]
I see no reason why it would be useful to someone having read-only access.

**Comment by georgehrke** [comment]
Related to https://github.com/nextcloud/calendar/issues/519

Things to fix:
- [ ] Do not show accessClass property in shared calendar
- [ ] If the user already set an accessClass, do not allow moving it to a calendar shared with them
- [ ] If the event is in a calendar shared with the user, and the accessClass is not PUBLIC, open the event in read-only mode.

@tcitworld Do you agree with all those points? :)

**Comment by tcitworld** [comment]
> If the event is in a calendar shared with the user, and the accessClass is not PUBLIC, open the event in read-only mode.

If the accessClass is not `PUBLIC` shouldn't anything at all be shown anyaway (apart from busy with `CONFIDENTIAL`) ?

**Comment by georgehrke** [comment]
Yes, Exactly. If the event is labeled as busy only, it should be read-only to the users it is shared with.
